wp-content/themes/oceanwp-child/bbpress/content-archive-topic.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * Archive Topic Content Part
+ *
+ * @package bbPress
+ * @subpackage Theme
+ */
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+?>
+
+<style>
+	.page-header{
+		display:none;
+	}
+
+	.content-area{
+		padding-right: 0px;
+    	border-right-width: 0px;
+	}
+</style>
+
+<div class="row mt-60">
+	<div class="col-md-4 col-lg-4 col-sm-12">
+		<h2>Discuss</h2>
+		<p>The Global Green Recovery Collaborative invites members to share ideas for a green recovery and working together, 
+			to guide one another and share resources. Connect with experts, practitioners, activists, and researchers.</p>
+		<p>Register and create your profile to participate. Frequently Asked Questions and Community Guidelines will 
+			answer any questions you have about joining and participating.</p>
+
```

Comment:
Can this be dynamic content? e.g. title and content?

Reply:
You want it to be dynamic? I thought these are static contents which shall not change once they provide us the final contents. In that case, we have to make other contents dynamic on other pages like initiative lists page, suggest initiative or other pages having contents?